apps/rush-lib/src/schemas/rush.schema.json

```diff
@@ -3,31 +3,44 @@
   "title": "Rush main config File",
   "description": "The main configuration file for the Rush multi-project build tool. See http://rushjs.io for details.",
   "type": "object",
-
+  "definitions": {
+    "environmentVariables": {
+      "description": "Enviroment variables for the package manager",
+      "type": "object",
+      "additionalProperties": {
+        "type": "object",
+        "properties": {
+          "value": {
+            "type": "string"
+          },
+          "override": {
+            "type": "boolean"
+          }
+        },
+        "additionalProperties": false
+      }
+    }
+  },
   "properties": {
     "$schema": {
       "description": "Part of the JSON Schema standard, this optional keyword declares the URL of the schema that the file conforms to. Editors may download the schema and use it to perform syntax highlighting.",
       "type": "string"
     },
-
```

Comment:
Can you put these blank lines back?

Reply:
VS Code automatically removed them when I saved the file. Should we maintain these blank lines?